### PR TITLE
Firefox for Android 79 does a lot of new stuff

### PR DESCRIPTION
**Pull request by atjn**
Following the release of Firefox 79: #5561

I've verified all the changes that were expected when going from Gecko 68 to Gecko 79. All changes in this PR have been manually tested with the [caniuse test](https://tests.caniuse.com/), unless otherwise noted in the commit message.

I used a prototype of a new script i'm working on, which should ensure that i have tested every single feature that might have changed since Gecko 68. It might've missed a few features that are mobile-specific, but other than that, this list is complete.

There's also a few other changes in the list - for things i stumbled upon while testing Firefox.

### CODE REVIEW

**Comment by Fyrd** [comment]
Looking good! Not seeing the update yet myself but once I do I'll review this.

**Comment by atjn** [comment]
Just stumbled across a feature that was added to mobile, but isn't in Firefox desktop. So as i said, i might have missed a few mobile-specific features in this list.

**Comment by atjn** [comment]
@Fyrd large parts of this PR have been reverted in b0ed49e

File: features-json/loading-lazy-attr.json [hunk 1]
```diff
@@ -131,14 +131,14 @@
...
-      "80":"y",
-      "81":"y",
-      "82":"y"
+      "75":"a #1",
+      "76":"a #1",
+      "77":"a #1",
+      "78":"a #1",
+      "79":"a #1",
+      "80":"a #1",
+      "81":"a #1"
```
Comment (Fyrd) [inline features-json/loading-lazy-attr.json:140]: @atjn Looks like the missing comma here prevented the data in this file from being updated. Not sure if the validator missed it or if I didn't pay enough attention to it...

**Comment by Fyrd** [comment]
@atjn Thanks for pointing that out! I believe I've fixed those parts now. Looks like it was mostly the `loading-lazy-attr` that was missing as well as a few links (when both the title and url changes for a link my updating code often treats it as a new link instead of an old one being updated, will need to fix that). If there's anything else specific I missed let me know.

File: features-json/loading-lazy-attr.json (hunk 1 above)
Comment (atjn) [reply features-json/loading-lazy-attr.json:140]: I see, thanks for checking that out

**Comment by atjn** [comment]
Awesome, thanks! As far as i can tell, you got everything :)

**Pull request merged**